scripts/fuzz_opt.py

```diff
@@ -1654,10 +1654,15 @@ def handle(self, wasm):
                 os.unlink(f)
 
         # Call run.py(), similarly to how ClusterFuzz does.
-        run([sys.executable,
-             os.path.join(self.clusterfuzz_dir, 'run.py'),
-             '--output_dir=' + os.getcwd(),
-             '--no_of_files=1'])
+        out = run([sys.executable,
+                   os.path.join(self.clusterfuzz_dir, 'run.py'),
+                   '--output_dir=' + os.getcwd(),
+                   '--no_of_files=1'])
+
+        # We should not see any mention of a wasm-opt error that caused a
+        # retry. On production ClusterFuzz this is not an error, but we do want
+        # to know about such issues, as they may be real bugs in wasm-opt.
+        assert 'retry' not in out, out
```

Comment:
Are `assert`s not executed on production ClusterFuzz infra?

Reply:
They are, but `fuzz_opt.py` is not run there, instead we run `clusterfuzz/run.py`. `fuzz_opt` is only for our local testing.

Reply:
Got it, thanks 👍